mrmustard/lab_dev/states/base.py

```diff
@@ -1043,3 +1058,38 @@ def __rshift__(self, other: CircuitComponent | Scalar) -> CircuitComponent | Bat
             elif result.wires.bra.modes == result.wires.ket.modes:
                 result = DM(result.wires.modes, result.representation)
         return result
+
+    @classmethod
+    def random(cls, modes, max_r=1.0):
+        r"""
+        generates random states with 0 displacement, using the random_symplectic funcionality
+        Args: "modes" are the modes where the state is defined on
+        Output is a Ket
+        """
+        # TODO: use __class_getitem_ and sample from broader probabilities
+        # TODO: random any gate
+
+        # generate a random ket repr.
+        # e.g. S = math.random_symplectic(dim) and then apply to vacuum
+        m = len(modes)
+        S = math.random_symplectic(m, max_r)  # change to a,a^dagger
```

Comment:
what do you mean by change to a,a^dagger?

Reply:
The symplectic transform after applying \Omega = 1/sqrt(2) * [[1,1],[-i,i]]

Reply:
I removed the comment (I totally agree with you that it is not a clarifying comment).